src/modules/storage/LedgerStorage.ts

```diff
@@ -2178,6 +2335,44 @@ export class LedgerStorage extends Storages
 
         return this.query(sql, [from, to]);
     }
+    /**
+   * Get the transaction count for an address
+   * @returns Returns the Promise. If it is finished successfully the `.then`
+   * of the returned Promise is called with the records
+   * and if an error occurs the `.catch` is called with an error.
+   */
+    public getTxCount(address: string): Promise<any[]> {
+        let sql =
+            `SELECT COUNT(*) as tx_count FROM 
+             (
+              SELECT COUNT(*)
+               FROM 
+                   transactions T INNER JOIN tx_outputs O ON(O.tx_hash = T.tx_hash)
+               WHERE 
+                   O.address = ?
+               GROUP BY 
+                   T.tx_hash) Tx`;
```

Comment:
Counts transactions that occur at that address only.
Must count to tx_inputs.
Refund transactions are excluded.

Reply:
resolved